spec/licensee/matchers/dice_matcher_spec.rb

```diff
@@ -80,4 +80,39 @@
       end
     end
   end
+
+  context 'confidence similarity match' do
+    module Licensee
+      module ContentHelper
+        alias max_delta_original max_delta
+        alias max_delta calculate_max_delta
+        public :max_delta
+      end
+    end
+
+    Licensee.licenses(hidden: true).each do |license|
+      next if license.pseudo_license?
+      context "the #{license.key} license" do
+        let(:content) { license.content }
+
+        nearest =
+          Licensee
+          .licenses(hidden: true)
+          .reject { |other| other.pseudo_license? || other.key == license.key }
+          .collect { |other| [other, license.similarity(other)] }
+          .max_by { |other_similarity| other_similarity[1] }
+        next if nearest[1] < 50
+
+        it "matches #{nearest[0].key}" do
+          Licensee.confidence_threshold = nearest[1].floor
+
+          matcher = Licensee::Matchers::Dice.new(file)
+          similars = matcher.licenses_by_similiarity.map { |s| s[0] }
+
+          expect(similars).to include nearest[0]
```

Comment:
What are you trying to test here? That potential licenses include licenses above the set confidence threshold?

Reply:
Yes.